templates/data/generic_component_data.yaml

```diff
@@ -25,30 +25,6 @@ templates:
                     global:
                       - "jobConfig_postsubmit"
                       - "disable_testgrid"
-      - to: ../prow/jobs/kyma/components/application-registry/application-registry-generic.yaml
```

Comment:
application-connectivity-certs-setup-job configuration (above) should also be removed